content/source/docs/cloud/vcs/bitbucket-server.html.md

```diff
@@ -1,20 +1,22 @@
 ---
 layout: "cloud"
-page_title: "Bitbucket Server - VCS Providers - Terraform Cloud"
+page_title: "Bitbucket Server and Data Center - VCS Providers - Terraform Cloud"
 ---
 
 
-# Configuring Bitbucket Server Access
+# Configuring Bitbucket Server/Data Center Access
 
-These instructions are for using Bitbucket Server for Terraform Cloud's VCS features. Bitbucket Server is the on-premise version of Bitbucket; [Bitbucket Cloud has separate instructions,](./bitbucket-cloud.html) as do the [other supported VCS providers.](./index.html)
+These instructions are for using Bitbucket Server/Data Center for Terraform Cloud's VCS features. Bitbucket Server/Data Center are the on-premises versions of Bitbucket; [Bitbucket Cloud has separate instructions,](./bitbucket-cloud.html) as do the [other supported VCS providers.](./index.html)
 
-Note that Bitbucket Server requires both OAuth authentication and an SSH key. The instructions below include SSH key configuration.
+Note that Bitbucket Server/Data Center requires both OAuth authentication and an SSH key. The instructions below include SSH key configuration.
 
 -> **Version note:** Terraform Cloud supports Bitbucket Server versions 4.9.1 and newer. HashiCorp does not test older versions of Bitbucket Server with Terraform Cloud, and they might not work as expected. Also note that, although we do not deliberately remove support for versions that have reached end of life (per the [Atlassian Support End of Life Policy](https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html)), our ability to resolve customer issues with end of life versions might be limited.
 
+-> **Bitbucket Data Center Support:** Please note that these instructions are equally applicable to both Bitbucket Server and Bitbucket Data Center. Bitbucket Data Center users will follow the same instructions as a user with Bitbucket Server **with version greater than v5.4.0**, omitting the installation of the Web Post Hooks add-on. You will still select **Bitbucket Server** as the VCS Provider type.
```

Comment:
Is this call out not sufficient? I don't love the `Bitbucket Server/Data Center` scattered throughout and wonder if this note on its own is enough to satisfy the needs of the addition. Presumably those that run BBDC as also aware that it is BitBucket Server and would find their way to this connection type without the modification to title and naming?

Reply:
I considered that, and am not particularly precious either way - I just wanted to make sure it was explicitly called out in at least one place for SEO/other discoverability purposes. Will gladly revert the other instances and leave it just in the title with this callout if others agree.

Reply:
What does navigation look like with that long of a title? 
![image](https://user-images.githubusercontent.com/586829/63378270-81471080-c357-11e9-8ae7-0ccc68b3b68c.png)

Reply:
ah, I guess the navigation link isn't being changed at the moment, so we'd look like this

![image](https://user-images.githubusercontent.com/586829/63378333-a6d41a00-c357-11e9-982c-6ec94f8b0741.png)

Reply:
I haven't seen it rendered but the sidebar has a good amount of padding around it, I don't expect it would be any wider than "Users, Teams, Organizations" below

Reply:
OK @caseylang take another look, I've reduced the changes now.

Reply:
![image](https://user-images.githubusercontent.com/586829/63380106-6aa2b880-c35b-11e9-9f01-cab4a1050327.png)

Keep in mind that your change currently does not modify the navigation link. To do that you would need to update the following as well: 
https://github.com/hashicorp/terraform-website/blob/b7b7e6c33c2f9092bb7620227207d47bf6785988/content/source/layouts/cloud.erb#L60-L62

Reply:
Good call, thanks - added to PR